assets/airports/eidw.json

```diff
@@ -1,1329 +1,5087 @@
-        "ASDER": ["N53d33m46.70", "W6d22m26.40"],
-        "AKIVA": ["N53d38m56.00", "W6d27m09.60"],
-        "EPIDU": ["N53d49m03.50", "W6d35m54.80"],
-        "ADNAL": ["N53d41m53.90", "W6d35m41.50"],
-        "APRUT": ["N53d41m49.00", "W6d45m34.90"],
-        "DW865": ["N53d38m42.80", "W6d53m58.40"],
-        "DW866": ["N53d33m29.10", "W6d58m27.20"],
-        "DW551": ["N53d24m44.50", "W6d06m31.40"],
-        "DW554": ["N53d24m38.80", "W6d04m51.60"],
-        "NEKIL": ["N53d31m16.60", "W6d40m23.60"],
-
-        "NIMAT": ["N53d57m54.10", "W5d44m31.70"],
-        "RONON": ["N53d42m33.90", "W6d36m19.20"],
-        "ORVEN": ["N53d39m53.50", "W6d11m29.80"],
-        "GIRAS": ["N53d38m21.00", "W5d57m33.20"],
-        "ADSIS": ["N53d41m03.10", "W5d39m34.00"],
-        "KERAV": ["N53d37m42.70", "W5d45m57.30"],
-        "KOGAX": ["N53d34m18.60", "W5d38m14.10"],
-        "KUDOM": ["N53d29m25.80", "W5d33m14.30"],
-        "DW814": ["N53d23m47.50", "W5d31m41.10"],
-        "DW815": ["N53d18m12.90", "W5d33m46.60"],
-        "DW816": ["N53d13m46.90", "W5d38m44.40"],
-        "NARMU": ["N53d26m43.20", "W5d51m34.30"],
-
-        "ABLIN": ["N52d46m58.00", "W4d59m33.00"],
-        "IRKUM": ["N52d59m48.00", "W5d22m39.00"],
-        "PEKOK": ["N53d07m39.30", "W5d34m00.80"],
-        "SIVNA": ["N53d11m52.30", "W5d38m27.70"],
-        "SUGAD": ["N53d17m22.50", "W5d31m39.80"],
-        "DW704": ["N53d24m03.70", "W5d29m10.10"],
-        "DW705": ["N53d30m46.60", "W5d31m26.40"],
-        "DW706": ["N53d36m21.30", "W5d38m06.90"],
-        "SOPEP": ["N53d21m05.90", "W5d52m29.40"],
-        "VATRY": ["N52d33m16.00", "W5d30m00.00"],
-        "KEPOR": ["N53d10m16.50", "W6d22m00.70"],
-        "ARVOK": ["N53d09m19.00", "W6d03m35.10"],
-        "SORIN": ["N53d08m29.30", "W5d48m22.50"],
-
-        "RUBAR": ["N53d08m08.90", "W5d48m05.40"],
-        "DETAX": ["N53d14m30.70", "W6d12m35.60"],
-        "BERMO": ["N53d17m38.90", "W6d24m50.80"],
-        "DIRUM": ["N53d00m09.70", "W6d39m40.00"],
-        "KANUS": ["N53d06m30.50", "W6d26m52.10"],
-        "BABON": ["N53d13m03.30", "W6d30m56.50"],
-        "BIVDI": ["N53d10m59.50", "W6d40m05.60"],
-        "DW754": ["N53d12m02.30", "W6d49m42.60"],
-        "DW755": ["N53d15m54.20", "W6d57m04.50"],
-        "OSLEX": ["N53d21m55.80", "W6d41m44.50"],
-
-        "IRDEX": ["N53d11m45.10", "W6d03m50.10"],
-        "ERVAD": ["N53d06m47.10", "W6d04m40.40"],
-        "DWS78": ["N53d14m06.00", "W6d25m04.00"],
-        "NITIL": ["N53d28m00.00", "W6d37m55.00"],
-        "DW555": ["N53d20m13.70", "W5d53m44.00"],
-        "NEPOD": ["N52d56m57.20", "W6d10m29.80"],
-
-        "RISAP": ["N53d26m27.40", "W6d38m13.20"],
-        "LAPMO": ["N53d24m11.00", "W5d56m44.10"],
-        "GANET": ["N53d26m06.50", "W6d31m33.80"],
-        "MAXEV": ["N53d24m33.50", "W6d03m17.30"],
-        "BAPDA": ["N53d37m40.00", "W6d24m03.70"],
-        "INTOP": ["N53d13m27.40", "W6d06m32.40"]
+    "SUROX(AB)": {
+      "icao": "SUROX(AB)",
+      "name": "Surox",
```

Comment:
Including the `(AB)` with the procedure, though not the ideal solution, will work for now, because it will be a while before we add different variations of the same procedure based on aircraft type, since that (so far) seems to be unique to EIDW.